app/models/ability.rb

```diff
@@ -79,6 +79,7 @@ def signed_in(user)
 
     can :index, Ticket
     can :manage, TicketPurchase, user_id: user.id
+    can [:index, :new, :create], Payment
```

Comment:
No, a user can only pay his tickets. I will repair the abilities to make it correct.